internal/plugin/resource_bundle_install.go

```diff
@@ -466,8 +466,8 @@ only one can be configured at a time.
 					Optional:        true,
 					DescriptionKind: tfprotov6.StringKindMarkdown,
 					Description: docCaretToBacktick(`
-- ^artifactory.username^ (String) The Artifactory API username. This will likely be your hashicorp email address
-- ^artifactory.token^ (String) The Artifactory API token. You can sign into Artifactory and generate one
+- ^artifactory.token^ (String) The Artifactory identity token. You can get one by joining the 'artifactory-users' Doormat
```

Comment:
Let's change our single quotes to be carets and they'll get interpreted into backticks in the rendered markdown.